src/app/static/src/store/actions.ts

```diff
@@ -1,29 +1,57 @@
-import axios from "axios";
-import { ActionTree } from "vuex";
+import axios, { AxiosError } from "axios";
+import { ActionTree, Commit } from "vuex";
 import { RootState } from "@/store/state";
-import { ParameterGroupJsonataMetadata } from "@/types";
+import { ErrorInfo, ParameterGroupJsonataMetadata } from "@/types";
 import jsonata from "jsonata";
 
+export function commitErrors(e: AxiosError, commit: Commit): void {
+    let errors: Array<ErrorInfo>;
+    if (e.response && e.response.data && e.response.data.errors) {
```

Comment:
```suggestion
    if (e.response?.data.errors) {
```
...if you wanted to make this slightly easier to read/write